topdeals/topdeals.html

```diff
@@ -14,7 +14,7 @@
 
         body {
             font-family: Arial, sans-serif;
-            background:url("bestdeals.jpg");
+            background:url("topdeals_bg.jpg");
```

Comment:
![image](https://github.com/user-attachments/assets/098eabe8-4920-40a7-a314-4e30da138cf8)

u have added some bg , but there is only a white bg its showing that , & there is alo an unusual blank space on the top of the page